embedded-workshop-book/src/advanced-next-steps.md

```diff
@@ -0,0 +1,33 @@
+# Next Steps
+
+## String descriptors
+
+Here's one more USB exercise for you: add string descriptors support to the USB firmware.
+
+- First, you'll want to read through section 9.6.7 of the USB spec, which covers string descriptors.
+- Next, we suggest you change your *configuration* descriptor to use string descriptors. You'll want to change the `iConfiguration` field to a non-zero value.
```

Comment:
wait but wouldn't that break enumeration? Or do you mean duplicate and change?

Reply:
yes, this will break enumeration. Fixing it to support string descriptors is the exercise.